compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -3324,6 +3325,13 @@ fn print_disambiguation_help<'tcx>(
         {
             let def_kind_descr = tcx.def_kind_descr(item.kind.as_def_kind(), item.def_id);
             let item_name = item.ident(tcx);
+            let first_arg_type = tcx
```

Comment:
Could you merge the two statements? Something like `let (first_arg_type, recv_ref) = …`? They share a lot.